cmd/drone-docker/main.go

```diff
@@ -238,6 +238,21 @@ func main() {
 			Usage:  "additional host:IP mapping",
 			EnvVar: "PLUGIN_ADD_HOST",
 		},
+		cli.BoolFlag{
+			Name:   "load-image",
+			Usage:  "load image from a file instead of building it",
+			EnvVar: "PLUGIN_LOAD_IMAGE",
+		},
+		cli.StringFlag{
+			Name:   "loaded-image-tag",
+			Usage:  "the tag of the loaded image",
+			EnvVar: "PLUGIN_LOAD_IMAGE_TAG",
```

Comment:
is it the tag of image loaded from file or is it tag not related to these two?
If it is unrelated then maybe it shouldn't be around there?

Reply:
So... the image yyyy... image [I'm referring to the .tar.gz that can be loaded] has a tag embedded in it, and I don't know how to extract that tag during plugin execution, so right now it needs to be explicitly passed [so that the plugin can operate on the loaded image as if it was builded in the default execution flow]. Does that answer your concern or did I misread something? I'm not completely sure I understand what you mean.